integration-tests/tracing-tests/pom.xml

```diff
@@ -66,6 +66,16 @@
     <dependency>
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>tracing-zipkin</artifactId>
+      <exclusions>
```

Comment:
should find the root cause of conflicts

Reply:
ok, I see

Reply:
**main reason**:
the version 1.7.7 do not have the check of log4j-over-slf4j.jar AND slf4j-log4j12.jar . 
But in the  version 1.2.36 , Log4jLoggerFactory will check it. 

**change**: 
  do not need to exclude all self4j.
```
    <dependency>
      <groupId>org.apache.servicecomb</groupId>
      <artifactId>tracing-zipkin</artifactId>
      <exclusions>
        <exclusion>
          <groupId>org.slf4j</groupId>
          <artifactId>log4j-over-slf4j</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

**more deatails**:
https://github.com/qos-ch/slf4j/commit/7a23ca74ae94f87675b25414d6c67a9e9ba9d98f#diff-8c10d2281ed0eaa6d7be5f600d9d673c
https://www.slf4j.org/codes.html#log4jDelegationLoop